src/test/java/com/hocs/test/pages/decs/Search.java

```diff
@@ -746,7 +762,11 @@ public void assertCOMPInformationRandomSearchResult(String criteria) {
                     workstacks.unallocateSelectedCase(randomSearchResult);
                     workstacks.selectSpecificCaseReferenceLink(randomSearchResult);
                 }
-                openOrCloseAccordionSection("Registration");
+                if (caseType.equalsIgnoreCase("COMP") || caseType.equalsIgnoreCase("IEDET")) {
```

Comment:
if the initial search is successful (we don't need to create a case), is the caseType session variable set, or is it still null?

Reply:
The caseType variable isn't set by calling the session variable, it's from splitting the randomSearchResult case reference and taking the substring prior to the first '/'. So caseType should always be set so long as the search returns a non-zero number of search results.